timemachine/fe/rbfe.py

```diff
@@ -707,8 +706,8 @@ def run_edge_and_save_results(
         mol_b = mols[edge.mol_b_name]
         core, smarts = atom_mapping.get_core_with_alignment(mol_a, mol_b, threshold=2.0)
 
-        complex_res, complex_top = run_complex(mol_a, mol_b, core, forcefield, protein, n_frames, seed)
-        solvent_res, solvent_top = run_solvent(mol_a, mol_b, core, forcefield, protein, n_frames, seed)
+        complex_res, complex_top = run_complex(mol_a, mol_b, core, forcefield, protein_path, n_frames, seed)
```

Comment:
run_complex/run_solvent will take either the path or the object. But the object is better for this use case.